Calculator/Program.cs

```diff
@@ -1,12 +1,21 @@
 ﻿using System;
 
-namespace Calculator
+namespace CalculatorOOPS
 {
-    class Program
+    public static class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            try
```

Comment:
performing try catch here is not responsibility of main method. move it inside the ReadMenuforCalcType method.